packages/hoppscotch-common/src/components/http/URLEncodedParams.vue

```diff
@@ -44,7 +44,7 @@
         />
       </div>
     </div>
-    <div v-if="bulkMode" class="h-full relative">
+    <div v-if="bulkMode" class="h-full relative w-full flex flex-1">
```

Comment:
It seems like the `flex-col` is usually applied across similar places although the direction doesn't matter much since it's just a single child.